lib/clowne/adapters/base.rb

```diff
@@ -71,6 +37,23 @@ def init_record(record)
   end
 end
 
-require 'clowne/adapters/base/init_as'
-require 'clowne/adapters/base/nullify'
-require 'clowne/adapters/base/finalize'
+Clowne::Adapters::Base.register_resolver(
+  :init_as,
+  Clowne::Resolvers::InitAs,
+  prepend: true
+)
+
+Clowne::Adapters::Base.register_resolver(
+  :nullify,
+  Clowne::Resolvers::Nullify
+)
+
+Clowne::Adapters::Base.register_resolver(
+  :finalize, Clowne::Resolvers::Finalize,
+  after: :nullify
+)
+
+Clowne::Adapters::Base.register_resolver(
+  :post_processing, Clowne::Resolvers::PostProcessing,
```

Comment:
Maybe, rename it to `after_commit`? (if we go with `commit` naming)

Reply:
Hm...it's better than `post_processing `but how do you think will people not be confused with AR callback? 🤔

Reply:
Yep, it could be.

Maybe, `after_clone`?

Reply:
`after_clone` very closest to `finalize` IMO :( Also not so readable: `after_clone do |origin, clone|`

Couple of Variants:

1. `fulfill_with_persisted { |origin, clone| .. }`
2. `with_persisted { |origin, clone| .. }`
3. `process_persisted { |origin, clone| .. }` 

Or something similar to `to fix` (because it is the main goal):
1. `repair { |origin, clone| ... }`
2. `overhaul { |origin, clone| ... }` 
3. `grind { |origin, clone| .. }`
4. `smooth { |origin, clone| .. }`